components/CategorySelection/index.js

```diff
@@ -24,12 +27,18 @@ const CategoriesSelection = () => {
       </div>
       <div className="container mx-auto">
         <div className="flex flex-wrap items-center justify-center font-bold">
-          {categories.data.map(({ id, name, slug }) => (
+          {categories.data.map(({ id, slug }) => (
             <Card
               key={id}
               id={id}
               title={convertSlugToTitle(slug)}
-              url={imgUrl}
+              url={
+                slug === "mental-health"
+                  ? mentalHealth
+                  : slug === "life-issues"
+                  ? lifeIssues
+                  : socialRelationship
+              }
```

Comment:
Evening Negin. This looks similar to some other code that we tidied up in another PR. This is mainly because nesting ternaries like this be pretty hard to read, at least that's what I find. I very rarely nest ternaries at all.

What I'd suggest here is making a small lookup Object. The keys of the object will be the slugs, and the values will be the imported variables, like this:

``` const urlLookup = { "mental-health": mentalHealth, "life-issues": lifeIssues...}```

This lookup object can just be a variable in this file.

Then all you do for the url is say ```url={urlLookup[slug]}``` and it'll put the relevant value in that prop. Does that make sense?

Reply:
Hi @ARTurnerGit,
 I swear I thought of changing it to a function :D but since the original code wasn't written by me I didn't do it. But using the object seem very reasonable, @oksygenn and I will do it tomorrow, Thank you for the explanation